### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by VSEphpbb**

### CODE REVIEW

**Comment by rxu** [comment]
Same as here https://github.com/phpbb-extensions/viglink/pull/31#issuecomment-227103735
:wink:

**Comment by VSEphpbb** [comment]
Updated

File: .travis.yml [hunk 1]
```diff
@@ -57,4 +56,4 @@ script:
   - sh -c "if [ '$SNIFF' != '0' ]; then travis/ext-sniff.sh $DB $TRAVIS_PHP_VERSION $EXTNAME; fi"
   - sh -c "if [ '$IMAGE_ICC' != '0' ]; then travis/check-image-icc-profiles.sh $DB $TRAVIS_PHP_VERSION; fi"
   - phpBB/vendor/bin/phpunit --configuration phpBB/ext/$EXTNAME/travis/phpunit-$DB-travis.xml --bootstrap ./tests/bootstrap.php
-  - sh -c "if [ '$EPV' != '0' ] && [ '$TRAVIS_PHP_VERSION' = '5.3.3' ] && [ '$DB' = 'mysqli' ]; then phpBB/ext/$EXTNAME/vendor/bin/EPV.php run --dir='phpBB/ext/$EXTNAME/'; fi"
+  - sh -c "if [ '$EPV' != '0' ] && [ '$TRAVIS_PHP_VERSION' = '5.5' ] && [ '$DB' = 'mysqli' ]; then phpBB/vendor/bin/EPV.php run --dir='phpBB/ext/$EXTNAME/'; fi"
```
Comment (marc1706) [inline .travis.yml]: Why 5.5? Aren't we using the min PHP verison which would be 5.4?

Comment (VSEphpbb) [inline .travis.yml]: This PR is for 3.1.x. This is just for consistency with all our other 3.1.x extensions. On the 3.2.x/master it will rely on NOTESTS.

**Pull request merged**